test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change_with_blacklist.test

```diff
@@ -0,0 +1,492 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Please take into account in this file the feedback provided to test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change.test (when applicable)

Reply:
fixed in f9887f1